hazelcast-client/src/test/java/com/hazelcast/client/map/querycache/ClientQueryCacheBasicTest.java

```diff
@@ -96,7 +96,7 @@ public void setup() {
         if (useNearCache) {
             clientConfig.addNearCacheConfig(new NearCacheConfig()
                     .setName(TEST_MAP_NAME)
-                    .setCacheLocalEntries(true));
+                    .setInvalidateOnChange(true));
```

Comment:
do you think this was a bug in a test which was not caught due weak validation in EE?